client/src/components/App/Content/LandingPage/adminLandingPage/UnusualCard/UnusualCard.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Tooltip, Typography, Divider } from '@material-ui/core';
+import WarningIcon from '@material-ui/icons/Warning';
+import NavigateBeforeIcon from '@material-ui/icons/NavigateBefore';
+
+import FilterRulesVariables from 'models/FilterRulesVariables';
+import statusToFilterConvertor from 'commons/statusToFilterConvertor';
+import FilterRulesDescription from 'models/enums/FilterRulesDescription';
+
+import useHoverStyles from '../useHoverStyles';
+import LoadingCard from '../LoadingCard/LoadingCard';
+import useStyles, { cardHeight, cardWidth } from './UnusualCardStyles';
+
+const unusualCompletesInvestigationsText = 'חקירות אשר הושלמו ללא מגעים';
+const unusualInProcessInvestigationsText = 'חקירות בטיפול מעל 4 שעות';
+
+const UnusualCard: React.FC<Props> = (props: Props): JSX.Element => {
+    const classes = useStyles();
+    const hoverClasses = useHoverStyles();
+
+    const { onCompleteClick, onInProcessClick, isLoading, unusualInProgressInvestigationsCount, unusualCompletedNoContactInvestigationsCount} = props;
+
+    return (
+        <LoadingCard isLoading={isLoading} width={cardWidth} height={cardHeight} className={classes.unusualCard}>
+            <div className={classes.unusualTitle}>
+                <WarningIcon className={classes.warningIcon}></WarningIcon>
+                <Typography className={classes.unusualTitleText}><b>לתשומת ליבך</b></Typography>
+            </div>
+            <Tooltip className={[classes.unusualCompleted, hoverClasses.whiteButtons].join(' ')} title={unusualCompletesInvestigationsText}>
```

Comment:
I think you should think over the use of hoverClasses.whiteButtons at this location, cursor: 'pointer' is being used and not all parts in your card are clickable

Reply:
removed cursor instead.